### PR TITLE
Update pyyaml to 6

### PR DESCRIPTION
**Pull request by normanrz**
Unfortunately, my use case wasn't solved with #9, because I need compatibility with pyyaml 6. This PR adds support for `>=5.4`

### CODE REVIEW

**Comment by TheWall89** [comment]
Ah, no worries. We can bump to 6. I just feared breaking changes between 5 and 6 and so I went for the safest choice. Will try to bump to 6 when I have time.

File: pyproject.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ keywords = ['openapi', 'rest']
 
 [tool.poetry.dependencies]
 python = ">=3.7.0, <4.0.0"
-pyyaml = "^5.4"
+pyyaml = ">=5.4"
```
Comment (TheWall89) [inline pyproject.toml:14]:
```suggestion
pyyaml = "^6.0"
```

I prefer to depend only on one major version, not both 5.4.x and 6.x.x. Two major versions can present breakinig changes.

**Comment by TheWall89** [comment]
Hi @normanrz . Your branch has conflicts. You need to merge master, fix conflict in `pyproject.toml` and re-generate `poetry.lock`. (also `poetry.lock` shows conflict but since it is autogenerated it is not worth fixing it manually)

If you don't have time, I'll create a new PR because I cannot add new commits to yours.

Thanks again for your contribution :)

**Comment by normanrz** [comment]
Thanks for pointing out the conflicts. I must have overlooked them.

**Review by TheWall89 — APPROVED** [review]

**Pull request merged**